### PR TITLE
gradle-profiler 0.12.0 (new formula)

### PR DESCRIPTION
**Pull request by JakeWharton**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gradle-profiler.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install %w[bin lib]
+    (bin/"gradle-profiler").write_env_script libexec/"bin/gradle-profiler",
+      :JAVA_HOME => "${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
+  end
+
+  test do
+    (testpath/"build.gradle").write <<~EOS
```
Comment (wolfs) [inline Formula/gradle-profiler.rb:20]: I think you should add an (empty) `settings.gradle` as well, so Gradle doesn't try to search upwards for one.

Comment (JakeWharton) [reply Formula/gradle-profiler.rb:20]: Can do. For what it's worth this is copied from the `gradle.rb` formula. I will fix it separately.

Comment (wolfs) [reply Formula/gradle-profiler.rb:20]: Awesome, thank you!

Comment (JakeWharton) [reply Formula/gradle-profiler.rb:21]: Fixed here and for Gradle in https://github.com/Homebrew/homebrew-core/pull/58482

File: Formula/gradle-profiler.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    libexec.install %w[bin lib]
+    (bin/"gradle-profiler").write_env_script libexec/"bin/gradle-profiler",
+      :JAVA_HOME => "${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
+  end
+
+  test do
+    (testpath/"settings.gradle").write ""
+    (testpath/"build.gradle").write <<~EOS
+      println "Hello"
+    EOS
```
Comment (SeekingMeaning) [inline Formula/gradle-profiler.rb:23]:
```suggestion
    (testpath/"build.gradle").write 'println "Hello"'
```

**Review by SeekingMeaning — APPROVED** [review]

**Comment by issyl0** [comment]
Thanks for your contribution to Homebrew, @JakeWharton!

**Pull request closed without merging**